PayPalDataCollector/src/androidTest/java/com/paypal/android/paypaldatacollector/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.paypal.android.paypaldatacollector
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
Should we remove this or update it to PayPalDataCollectorTest with a TODO to write integration tests?